src/app/components/account/bitcoin-account-loader.tsx

```diff
@@ -0,0 +1,21 @@
+import { useConfigBitcoinEnabled } from '@app/query/common/remote-config/remote-config.query';
+import { useCurrentAccountIndex } from '@app/store/accounts/account';
+import { useNativeSegwitSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+interface BitcoinAccountLoaderProps {
+  index?: number;
+  current?: true;
+  children: (address: string) => React.JSX.Element;
+}
```

Comment:
I would break this up like with the example I sent with the discriminating union.

`<BitconAccountLoader index={43} current />` is a valid type and shouldn't be